install_octopus.sh

```diff
@@ -79,20 +79,3 @@ cat octopus-configlog-warnings
 # all in one line to make image smaller
 make -j && make install && make clean && make distclean
 
-
-
-if [ $version == "develop" ]; then
-  # Set ENV variable for external libs (only needed for octopus14.0 onwards)
-  echo "Section Issue 9 starts here. --------------"
-  echo "Issue 9: https://github.com/fangohr/octopus-in-docker/issues/9"
-  # DEBUG output
-  ldd /usr/local/bin/octopus | grep libsym
```

Comment:
While we remove the ldd check here. It might be useful to print the ldd output in the CI. Might be something we would want to do in some other MR/issue.